test/ca_bundle/ca_bundle_test.go

```diff
@@ -29,20 +29,21 @@ const (
 	commonConfigTOML       = "/common-config.toml"
 	targetString           = "x509: certificate signed by unknown authority"
 
-	// Let the agent run for 30 seconds. This will give agent enough time to call server
-	agentRuntime         = 30 * time.Second
 	localstackS3Key      = "integration-test/ls_tmp/%s"
 	keyDelimiter         = "/"
 	localstackConfigPath = "../../localstack/ls_tmp/"
 	originalPem          = "original.pem"
 	combinePem           = "combine.pem"
 	snakeOilPem          = "snakeoil.pem"
 	tmpDirectory         = "/tmp/"
+	runEMF               = "sudo bash resources/emf.sh"
```

Comment:
I believe this should be a file instead of built into the ami like we assume in our other tests